configuration.nix

```diff
@@ -34,7 +34,7 @@
   # Enable this module to use spark-wallet, a minimalistic wallet GUI for
   # c-lightning, accessible over the web or through mobile and desktop apps.
   # Only enable this if clightning is enabled.
-  # services.spark-wallet.enable = true;
+  services.spark-wallet.enable = true;
```

Comment:
Why are we enabling spark-wallet by default?

Reply:
fixed